.github/workflows/CrossRefCheckWithAnchors.yml

```diff
@@ -2,11 +2,9 @@ name: CrossRefCheck with Anchors
 
 on: 
   push:
-    branches:
-      - review
 
 jobs:
   crossref-check-with-anchors:
     uses: Open-Systems-Pharmacology/Workflows/.github/workflows/MarkdownCrossrefCheck_withAnchors.yml@main
     with:
-      folder-path: 'Evaluation/report'
+      folder-path: 'Qualification/report'
```

Comment:
the check will fail now because of the missing images. As soon as this copy image problem is fixed - the check should pass.